src/components/dashboard/GapByRace.vue

```diff
@@ -58,44 +144,59 @@ const activeStats = computed(() => {
   );
   if (row) {
     // don't let a population be more than 100% vaccinated
-    return fieldData.value.map((f) => {
-      const population = row[f.populationField];
-      return {
-        name: f.name,
-        pct:
-          population > 0
-            ? Math.min(1, row[f.observedField] / row[f.populationField])
-            : 0,
-        gap: Math.max(0, row[f.expectedField] - row[f.observedField]),
-        population,
-      };
-    });
+    return fieldData.value
+      .map((f) => {
+        const population = row[f.populationField];
+        return {
+          name: f.name,
+          pct:
+            population > 0
+              ? Math.min(1, row[f.observedField] / row[f.populationField])
+              : NaN,
+          gap: Math.max(0, row[f.expectedField] - row[f.observedField]),
+          population,
+        };
+      })
+      .sort(sortByProperty("pct"));
```

Comment:
ah, I see it is sorted

instead of referring to the 0th item everywhere, I still think I'd pull that one out to a computed value that takes the 0th item and is named more meaningfully

Reply:
I've updated the `minVaxRace` variable